Dockerfile.rocm

```diff
@@ -1,15 +1,16 @@
 # default base image
-ARG BASE_IMAGE="rocm/pytorch:rocm6.0_ubuntu20.04_py3.9_pytorch_2.1.1"
+ARG BASE_IMAGE="rocm/pytorch:rocm6.1_ubuntu22.04_py3.10_pytorch_2.1.2"
 
 FROM $BASE_IMAGE
 
-ARG BASE_IMAGE="rocm/pytorch:rocm6.0_ubuntu20.04_py3.9_pytorch_2.1.1"
+ARG BASE_IMAGE="rocm/pytorch:rocm6.1_ubuntu22.04_py3.10_pytorch_2.1.2"
```

Comment:
QQ: Why are we defining this arg twice?

Reply:
because, the first BASE_IMAGE is used the `FROM $BASE_IMAGE` statement. And after the FROM statement, it no longer exists. We need to do it again in order to use it.

Reply:
Got it. Thanks!

Reply:
A second `ARG BASE_IMAGE` should exist after the `FROM` but it does not need to be defined. By default it imports the value of `BASE_IMAGE` from before the build stage.